app/controllers/api/v1/bathrooms_controller.rb

```diff
@@ -20,8 +22,24 @@ def show
           }
         end
 
+
         render json: { status: 'SUCCESS', message: 'Loaded bathrooms', bathrooms: bathroom, reviews: parsed_reviews }, status: :ok
       end
+
+      def create
+        bathroom = current_user.bathrooms.new(bathroom_params)
```

Comment:
Can any users create a new bathroom or do you just want this limited to Users in your database?

Reply:
We haven't gotten to authorization stuff yet. Just wanted to get it working for anyone first